users/serializers.py

```diff
@@ -1,11 +1,16 @@
 from django.contrib.auth.password_validation import validate_password
+from drf_extra_fields.fields import Base64ImageField
 from rest_framework import serializers
 
 from users.models import User
 
 
 class UserPublicSerializer(serializers.ModelSerializer):
     id = serializers.IntegerField(read_only=True)
+    profile_photo = Base64ImageField(
+        max_length=None,
+        use_url=True,
```

Comment:
Not sure we need to set `use_url` explicitly as it is `True` by default
```
https://www.django-rest-framework.org/api-guide/fields/#filefield

Defaults to the value of the UPLOADED_FILES_USE_URL settings key, which is True unless set otherwise.
```

Reply:
Removed